server/migrations/20200903090107_update_pushes_default_branch_name.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex) {
+	return knex.schema.alterTable('pushes', t => {
+		t
+			.string('branch')
+			.defaultTo('trunk')
+			.alter();
```

Comment:
The [documentation for `alter()`](https://knexjs.org/#Schema-alter) says we need to repeat the `.notNullable()` declaration in order to keep it on the `branch` column. (And the `branch` column [is not-nullable](https://github.com/Automattic/icfy/blob/master/server/migrations/20171219225520_add_branch_column.js#L5).) In other words, the `alter()` operation is "replace" rather than "patch".

Maybe we also need to say `.string( 'branch', 200 )` to prevent resetting the VARCHAR size?